guardian/gateway
# Test adding different newsletters into the onboarding flow

## What does this change?


This sets up an A/B test where 50% of users will see different newsletter options when going through the onboarding flow.

## How to test


Set a `GU_mvt_id` cookie to `1` to see the changes being tested. Set it to `2` to see the control group behaviour, which shows the same list as is currently used.
Still looking into how best to test region-specific behaviour.

## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
